powerauth-java-client-axis/src/main/resources/soap/wsdl/serviceV3.wsdl

```diff
@@ -1510,6 +1516,19 @@
                 </xs:restriction>
             </xs:simpleType>
 
+            <xs:simpleType name="PlatformType">
+                <xs:annotation>
+                    <xs:documentation>Enum representing user device platforms.
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:restriction base="xs:string">
+                    <xs:enumeration value="IOS"/>
+                    <xs:enumeration value="ANDROID"/>
+                    <xs:enumeration value="HW"/>
+                    <xs:enumeration value="UNKNOWN"/>
+                </xs:restriction>
```

Comment:
Do not limit platform to the predefined set of platforms.

Reply:
Ok, I will make a String out of the platform. I will just apply consistent case to avoid inconsistent values, e.g. `iOS` and `ios`